web/src/pages/api/public/metrics/daily.ts

```diff
@@ -47,73 +47,47 @@ export default withMiddlewares({
               SUM(o.completion_tokens) "outputUsage",
               SUM(o.total_tokens) "totalUsage",
               COALESCE(SUM(o.calculated_total_cost), 0)::DOUBLE PRECISION as "totalCost"
-            FROM
-              traces t
-            LEFT JOIN observations_view o ON o.trace_id = t.id AND o.project_id = t.project_id
+            FROM traces t
+            LEFT JOIN observations_view o
+            ON o.trace_id = t.id
+            AND o.project_id = t.project_id
             WHERE o.start_time IS NOT NULL
+              AND o.project_id = ${auth.scope.projectId}
               AND t.project_id = ${auth.scope.projectId}
               ${traceNameCondition}
               ${userCondition}
               ${tagsCondition}
               ${fromTimestampCondition}
               ${toTimestampCondition}
-            GROUP BY
-              1,
-              2
-            ORDER BY
-              1,
-              2
```

Comment:
Order by should be irrelevant to the grouping in daily_model_usage (aside from a potentially different order in the aggregated json array). IMO that should be negligible as users can still match by model key. We can revert this if necessary, though.

Reply:
agree